src/core/src/EnvironmentOptions.C

```diff
@@ -264,4 +149,174 @@ std::ostream& operator<<(std::ostream& os, const EnvOptionsValues & obj)
   return os;
 }
 
+
+void
+EnvOptionsValues::set_defaults()
+{
+  m_help = UQ_ENV_HELP;
+  m_numSubEnvironments = UQ_ENV_NUM_SUB_ENVIRONMENTS_ODV;
+  m_subDisplayFileName = UQ_ENV_SUB_DISPLAY_FILE_NAME_ODV;
+  m_subDisplayAllowAll = UQ_ENV_SUB_DISPLAY_ALLOW_ALL_ODV;
+  m_subDisplayAllowInter0 = UQ_ENV_SUB_DISPLAY_ALLOW_INTER0_ODV;
+  //m_subDisplayAllowedSet  = ;
+  m_displayVerbosity = UQ_ENV_DISPLAY_VERBOSITY_ODV;
+  m_syncVerbosity = UQ_ENV_SYNC_VERBOSITY_ODV;
+  m_checkingLevel = UQ_ENV_CHECKING_LEVEL_ODV;
+  m_rngType = UQ_ENV_RNG_TYPE_ODV;
+  m_seed = UQ_ENV_SEED_ODV;
+  m_platformName = UQ_ENV_PLATFORM_NAME_ODV;
+  m_identifyingString = UQ_ENV_IDENTIFYING_STRING_ODV;
+  m_numDebugParams = UQ_ENV_NUM_DEBUG_PARAMS_ODV;
```

Comment:
Why aren't debug parameters parsed from the input file?  Should I add them?

Reply:
These parameters appear to be an atavism that haven't actually been *used* for anything:
* since 2010, at the latest
* in the version control master, rather than a branch
* in git, rather than svn

So our second best option here would be "dig through ancient svn branches to figure out what these are for", and our best would be "just get rid of them".

Reply:
Okay, if @dmcdougall agrees, I can purge them.

Reply:
I'm ok with this.